products/nexus.md

```diff
@@ -19,15 +19,15 @@ auto:
 
 releases:
 -   releaseCycle: "3"
-    releaseDate: 2016-10-03
+    releaseDate: 2017-04-07
     eol: false
     latest: "3.44.0"
     latestReleaseDate: 2022-12-19
     link: https://help.sonatype.com/repomanager3/product-information/release-notes
 
 -   releaseCycle: "2"
     # closest date I found is for 2.0.6 on https://help.sonatype.com/repomanager2/download/download-archives---repository-manager-2#DownloadArchivesRepositoryManager2-NexusProfessional2.0.6
-    releaseDate: 2012-06-25
+    releaseDate: 2017-04-07
```

Comment:
Wrong dates (because 2.0.0 and 3.0.0 are missing from in https://github.com/endoflife-date/release-data/blob/main/releases/nexus.json).